### PR TITLE
fix: readme bugs

### PR DESCRIPTION
**Pull request by 318097**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -7,14 +7,16 @@ The easiest way to get started with enquirer is to pass a [question object](#pro
 ```js
 const { prompt } = require('enquirer');
 
-const response = await prompt({
-  type: 'input',
-  name: 'username',
-  message: 'What is your username?'
-});
+async function start() {
```
Comment (jonschlinkert) [inline docs/usage.md]: What's the reason for this change? Earlier in the readme it states that awaited prompts need to be in async functions.

File: docs/footer.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Please see [CHANGELOG.md](CHANGELOG.md).
 
 ## ❯ Performance
 
+### System under Test
```
Comment (jonschlinkert) [inline docs/footer.md]: Please change to `System specs`

File: docs/custom-prompts.md [hunk 3]
```diff
@@ -4,14 +4,17 @@ With Enquirer 2.0, custom prompts are easier than ever to create and use.
...
 **How do I create a custom prompt?**
 
-Custom prompts are created by extending Enquirer's `Prompt` class, or one of the built-in [prompts](#-prompts) or low-level [types](#-types).
+Custom prompts are created by extending either:
+- Enquirer's `Prompt` class
+- one of the built-in [prompts](#-prompts), or 
+- low-level [types](#-types).
 
-<!-- Example: HaiKarate Custom Prompt -->
+<!-- Example: MyCustomPrompt Custom Prompt -->
```
Comment (jonschlinkert) [inline docs/custom-prompts.md]: This is a joke about [Hai Karate](https://en.wikipedia.org/wiki/Hai_Karate) cologne. Please revert these changes, but I like the other changes.

File: docs/usage.md (hunk 1 above)
Comment (318097) [reply docs/usage.md]: Yes, but I thought having a complete example might be good, where a user can just copy paste everything & run it.

File: docs/custom-prompts.md (hunk 3 above)
Comment (318097) [reply docs/custom-prompts.md]: I'm sorry. I don't know about that. 'MyCustomPrompt' was generic.
Will revert back.
